examples/common/fdiff.py

```diff
@@ -7,20 +7,18 @@
 """Monitor rate of change of loss."""
 from __future__ import annotations
 
-from typing import Any, Dict
-
 from composer.core import Callback, State
 from composer.loggers import Logger
 
 
 class FDiffMetrics(Callback):
     """Rate of chage of metrics.
 
-    tracks and plots the rate of change of metrics effectively taking the numerical
-    derivative of the metrics
+    tracks and plots the rate of change of metrics effectively taking the
+    numerical derivative of the metrics
     """
 
-    def __init__(self, diff_train_metrics=True, diff_eval_metrics=True):
+    def __init__(self, diff_train_metrics=False, diff_eval_metrics=True):
```

Comment:
unintentional default change im guessing?

Reply:
diff_train_metrics are super high variance making it almost impossible to get a real signal from fdiff.
It would effectively require a moving window or avg, but thats a bit out the scope of this pr.